mediabridge/data_processing/wiki_to_netflix.py

```diff
@@ -4,12 +4,15 @@
 import time
 from tqdm import tqdm
 import sys
+from db.insert_data import insert_into_mongo
```

Comment:
Test is failing:

```
ModuleNotFoundError: No module named 'db'
```

Should be `mediabridge.db.insert_data`. But actually, are you using this in this file?

Reply:
Just updated, the import is to be able to do: data = process_data(test = test) but I don't think insert_data exists anymore. I don't remember who did the process_data(test = test) part so I'm not 100% sure what it does.

Reply:
Okay, so `process_data` is a function in this file. But I don't understand why you're worried about that function, because this import is for `insert_into_mongo`. We can quickly search this file (`wiki_to_netflix.py`) and see that it's not being used, so it can safely be removed.